### PR TITLE
Use test->test dependencies from mundane

### PR DESCRIPTION
**Pull request by etorreborre**
Proof of concept for using `test->test` dependencies from `mundane`.

### CODE REVIEW

File: project/depend.scala [hunk 1]
```diff
@@ -5,14 +5,15 @@ object depend {
...
+  val mundaneVersion = "1.2.1-20151104055659-410bb41"
   val mundane = Seq(
       "com.ambiata" %% "mundane-io"
     , "com.ambiata" %% "mundane-control").map(_ % mundaneVersion) ++ Seq(
       "com.ambiata" %% "mundane-io" % mundaneVersion % "test->test")
 
   val mundaneTesting = Seq(
-      "com.ambiata" %% "mundane-testing"   % mundaneVersion)
+      "com.ambiata" %% "mundane-testing"   % mundaneVersion % "test->test",
+      "com.ambiata" %% "mundane-testing"   % mundaneVersion % "compile->test")
```
Comment (charleso) [inline project/depend.scala]: Why do we need this one?

Comment (etorreborre) [inline project/depend.scala]: Because for now `saws-testing` also has test classes in `src/main`. Once the `mundane` PR is merged I will fix that as well.

**Comment by etorreborre** [comment]
@charleso I have now moved the saws test classes to the `test` directory (similarly to `mundane` test classes).

**Comment by charleso** [comment]
Seems legit. Surface :+1:

**Pull request merged**